Makefile

```diff
@@ -178,6 +178,8 @@ ALL_GRPC_BACKENDS+=backend-assets/grpc/rwkv
 ALL_GRPC_BACKENDS+=backend-assets/grpc/whisper
 ALL_GRPC_BACKENDS+=backend-assets/grpc/local-store
 ALL_GRPC_BACKENDS+=$(OPTIONAL_GRPC)
+# Use filter-out to remove the specified backends
+ALL_GRPC_BACKENDS := $(filter-out $(REMOVE_GRPC_BACKEND),$(ALL_GRPC_BACKENDS))
```

Comment:
```suggestion
ALL_GRPC_BACKENDS := $(filter-out $(SKIP_GRPC_BACKEND),$(ALL_GRPC_BACKENDS))
```

Reply:
maybe a better name for it can be `SKIP_GRPC_BACKEND`? anyhow, thanks!

Reply:
indeed, good suggestion thanks